### DISCUSSION TITLE
當AI知道你還有XX年可以活

### DISCUSSION THREAD

**Discussion by BeaGle0928** [1101-GS4719-C/Python-Community-GS4719-C#1808] [General]
丹麥的科學家開發了一款名叫「life2vec」的演算法。這項系統可以推估人類的預期壽命，並且準確度竟然高達78%！他們從2008年到2020年間，針對600萬名丹麥人進行調查，將生活中發生的各種事件編設編號，藉此來分類受試者經歷的事件。
我認為他們應該是以受試者生活中的資訊來進行判斷，像是是否有前臂骨折、產後出血等，受試者不論是加薪、從城市搬到鄉下居住、心臟病發等等事件都會被清楚記錄，但這也可能會關乎到隱私權的問題。透過研究發現，人類過早死亡的原因可能包含：身為男性、接受過心理治療、從事技術職業；而相對來說，高收入者、擔任領導職務等則與長壽有關。
我覺得過早死亡的原因裡面包含了「身為男性」這項很好笑，原本以為大家常看到男生比較容易做一些蠢事只是個迷因，但沒想到竟然是經過科學研究的！我想這項研究可以知道能讓人更長壽的辦法以及有哪些關鍵因素。

**Comment by Patsy17** [comment]
好酷～不過意外事件永遠無法被算出
讓我想到以前有一部電影可以算出自己的死期是什麼時候
然後他們就會很珍惜最後的時光
